flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/Constants.java

```diff
@@ -105,4 +103,10 @@
     public static final String POD_TEMPLATE_DIR_IN_POD = "/opt/flink/pod-template";
     public static final String POD_TEMPLATE_CONFIG_MAP_PREFIX = "pod-template-";
     public static final String POD_TEMPLATE_VOLUME = "pod-template-volume";
+
+    // Kubernetes start scripts
+    public static final String KUBERNETES_JOB_MANAGER_SCRIPT_PATH =
+            "$FLINK_HOME/bin/kubernetes-jobmanager.sh";
```

Comment:
```suggestion
            "kubernetes-jobmanager.sh";
```
This _should_ work because `$FLINK_HOME/bin` is added to `PATH` (https://github.com/apache/flink-docker/blob/master/1.12/scala_2.11-java11-debian/Dockerfile#L54).